### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_NewDatabase_ScreenReader: ORCA Screen reader is not announcing selected or non selected state of list item from the dropdown.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#172]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG1.3.1; #Linux; #Element:EditField; #ScreenReader; #ORCA; #External; #External:AzureDataStudio; #FirstParty:23182;

### Actual Result:​
ORCA Screen reader is not announcing selected or non selected state of list item from the "Charset" and "Collation" dropdown.

### Expected Result:​
ORCA Screen reader should announce selected or non selected state of list item from the "Charset" and "Collation" dropdown.
For example we have selected "utf8mb4" list item then screen reader should announce it as "Charset utf8mb4 selected".

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to connected server and right click on it and TAB to "Manage" and hit ENTER key.
4. TAB to "New Dashboard" and hit ENTER key.
5. "New Dashboard" screen gets open. Start the ORCA screen reader.
6. TAB to "Charset" and "Collation" dropdown and hit ENTER key.
7. Observe that ORCA Screen reader is not announcing selected or non selected state of list item from the dropdown.

### User Impact:
Screen reader users will get confused about the announcement as they might have selected any list item but it is not announcing as selected.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:
[Github issue for selected or non selected state.webm](https://user-images.githubusercontent.com/90901519/228767830-4a1c5bd9-6665-4fd7-847f-4d37e54e5f7f.webm)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by rishky-msft** [comment]
Hi @PoojaNamde,

I have fixed the bug in attached PR.

You can download the pre-release extension and test it out.
https://github.com/microsoft/azuredatastudio-mysql/releases/download/v0.2.9-alpha.1/azuredatastudio-mysql-0.2.9-alpha.1.vsix

Steps for installing pre-release extension:
1. Uninstall the MySQL Extension if already installed. (Open ADS -> Extensions -> MySQL -> uninstall -> Restart ADS)
2.  Download the pre-release extension vsix from above link.
3.  Open ADS -> File -> Install Extension via VSIX File -> Select the dowloaded vsix.
4.  Pre release extension will be installed.

**Comment by PoojaNamde** [comment]
Verified this issue in the latest environment
Version: 1.45.0-insider 
Commit: 4329d04da3b86689556d4dceab7d9b3d5cde3f58
Date: 2023-05-15T05:30:26.887Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2

issue is not fixed as screen reader is not announcing selected or unselected.

**Comment by rishky-msft** [comment]
Hi @PoojaNamde,

As we discussed, this issue is common for all the dropdowns in Azure Data Studio.
So this is external to mysql extension.

If you dont find any exisiting related bug in azure data studio repo, please add a new issue in azure data studio repo.
Before raising the issue to azuredatastudio, Please also confirm if this is not a issue in other ubunut apps also.

**Comment by PoojaNamde** [comment]
Logged External bug: https://github.com/microsoft/azuredatastudio/issues/23182

As per the [External Bug Process - First Party Dependencies](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fGM4NTQxMGIxZGFhYjQwMTZmNTIzMDhkYWM5NDgyMWQwfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwNDM2MDg4Njg1MjIxNDZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkyT0RjMk5EQTROVFkzT1RzeE5qWTROelkwTURnMU5qYzVPekU1T2pjelpqUTNOV00zTFRnNU5Ea3RORFprT1MxaFlURTVMVGN3WlRobU9EWm1ZVGszT0Y4M1lqbG1ZVFV5TmkweVptRXhMVFF5WmpRdFltVXhOeTFsTXpRME5EbGpabVkyTlRSQWRXNXhMbWRpYkM1emNHRmpaWE09fDA5YTBhOTU1NzM0MDQ5NTM1OTJiMDhkYWM5NDgyMWQwfGZlNmZiOTdlMzBhNTRjNzJhNTRhNTU0Y2RjNDg0NzM3&sdata=ZVRCdGg0WldPVlRVQ2RveFFNV1FJUEc0bHc1a2QvWTExbkVLS3V5emQzYz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-ramakatare%40microsoft.com) assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-06-23;

Closing this bug as per the discussion with SME.
